packages/www/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# To be run from the root of the repository so that we can pick up design-system too
```

Comment:
Maybe a note that this is a development image only? Or is it not?

I guess it could be explicited in the filename as well/instead? Like `Dockerfile.dev` or `dev.Dockerfile`, idk 🤔 
Not necessary anyway, just a nit.

Reply:
Probably not terribly useful outside of development yet, but I'd like livepeer-in-a-box to be useful outside of dev, so that'd make this a non-development dependency I suppose.